Missing directory for `run-paper-benchmarks`

The `README.md` states:

> In order to reproduce the benchmark figures from the paper, execute the target make run-paper-benchmarks. This will download all the results of the benchmarks run from using make all, unpacks them and then runs the benchmark target. This will only require ~50 GB free disk space and can be run on a consumer-grade laptop.

When I run `make run-paper-benchmarks`, I get the following error:

```
$ make run-paper-benchmarks
wget https://zenodo.org/records/14679713/files/pheval-paper-results.tar.gz -P /Users/hansenp/development/pheval-paper/target
--2025-12-10 16:15:39--  https://zenodo.org/records/14679713/files/pheval-paper-results.tar.gz
Resolving zenodo.org (zenodo.org)... 188.185.43.153, 188.185.48.75, 137.138.52.235
Connecting to zenodo.org (zenodo.org)|188.185.43.153|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 5580067516 (5.2G) [application/octet-stream]
Saving to: ‘/Users/hansenp/development/pheval-paper/target/pheval-paper-results.tar.gz’

pheval-paper-results.tar.gz         100%[===================================================================>]   5.20G  18.9MB/s    in 10m 10s 

2025-12-10 16:25:49 (8.72 MB/s) - ‘/Users/hansenp/development/pheval-paper/target/pheval-paper-results.tar.gz’ saved [5580067516/5580067516]

tar -zxf /Users/hansenp/development/pheval-paper/target/pheval-paper-results.tar.gz --strip-components=1 -C /Users/hansenp/development/pheval-paper/target/results
tar: could not chdir to '/Users/hansenp/development/pheval-paper/target/results'

make: *** [install-paper-results] Error 1
```

I can fix the error by adding an `mkdir` to the `install-paper-results` section of the Makefile:

```
install-paper-results:
	wget https://zenodo.org/records/14679713/files/pheval-paper-results.tar.gz -P $(TARGET)
	mkdir -p $(RESULTS_DIR) # Added by Peter H.
	tar -zxf $(TARGET)/pheval-paper-results.tar.gz --strip-components=1 -C $(RESULTS_DIR)
```

--- a/Makefile
+++ b/Makefile
@@ -57,7 +57,7 @@ benchmark: run-phenotype-only-benchmarks run-variant-benchmarks run-structural-v
 
 # Downloads the results of the 'install-tools' and 'run' targets from Zenodo, unpacks them and then runs the 'benchmark'
 # target to produce Figure 3. of the paper
-run-paper-benchmarks: install-paper-results	benchmark
+run-paper-benchmarks: download-pheval-paper-corpora install-paper-results benchmark
 
 # Download and results of the analyses run for the PheVal paper and then run the benchmark target to produce Figure 3.
 # This will download a 5.3 GB archive which will expand to ~35 GB

--- a/Makefile
+++ b/Makefile
@@ -62,5 +62,6 @@ run-paper-benchmarks: install-paper-results	benchmark
 # Download and results of the analyses run for the PheVal paper and then run the benchmark target to produce Figure 3.
 # This will download a 5.3 GB archive which will expand to ~35 GB
 install-paper-results:
+	mkdir -p $(RESULTS_DIR)
 	wget https://zenodo.org/records/14679713/files/pheval-paper-results.tar.gz -P $(TARGET)
 	tar -zxf $(TARGET)/pheval-paper-results.tar.gz --strip-components=1 -C $(RESULTS_DIR)

--- a/src/corpora/corpora.mk
+++ b/src/corpora/corpora.mk
@@ -1,6 +1,7 @@
 
 # Download the pheval paper corpora
 download-pheval-paper-corpora:
+	mkdir -p $(CORPORA_DIR)
 	wget https://zenodo.org/records/14679713/files/pheval-paper-corpora.tar.gz -P $(TARGET)
 	tar -zxf $(TARGET)/pheval-paper-corpora.tar.gz -C $(TARGET)
 	wget https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/NA12878_HG001/latest/GRCh37/HG001_GRCh37_1_22_v4.2.1_benchmark.vcf.gz  -P $(CORPORA_DIR)

--- a/src/tools/aim.mk
+++ b/src/tools/aim.mk
@@ -19,6 +19,7 @@ install-ai-marrvel: venv
 run-ai-marrvel: VENV_NAME=aim
 run-ai-marrvel: venv
 	$(PIP) install pheval.ai_marrvel==0.1.9
+	mkdir -p $(RESULTS_DIR)/$(AIM_NAME)/phenopacket_store_0.1.11_variants
 	$(VENV_BASE)/bin/pheval run --input-dir "$(AIM_DIR)" \
 		--testdata-dir "$(CORPORA_DIR)/phenopacket_store_0.1.11_variants" \
 		--output-dir "$(RESULTS_DIR)/$(AIM_NAME)/phenopacket_store_0.1.11_variants" \

